src/output/plugin-loader.cpp

```diff
@@ -271,3 +246,41 @@ void plugin_manager::load_static_plugins()
     init_plugin(loaded_plugins["_focus"]);
     init_plugin(loaded_plugins["_close"]);
 }
+
+std::vector<std::string> wf::get_plugin_paths()
+{
+    std::vector<std::string> plugin_prefixes;
+    if (char *plugin_path = getenv("WAYFIRE_PLUGIN_PATH"))
+    {
+        std::stringstream ss(plugin_path);
+        std::string entry;
+        while (std::getline(ss, entry, ':'))
+        {
+            plugin_prefixes.push_back(entry);
+        }
+    }
+
+    plugin_prefixes.push_back(PLUGIN_PATH);
+
+    return plugin_prefixes;
+}
+
+std::string wf::get_plugin_path_for_name(std::vector<std::string> plugin_paths,
```

Comment:
I think it would be best to return std::optional<std::string> to indicate that this function may indeed fail.

Reply:
Done